lambda/src/main/scala/pricemigrationengine/model/BillingPeriod.scala

```diff
@@ -13,11 +13,11 @@ object BillingPeriod {
   val notificationPaymentFrequencyMapping = Map(
     // This map is used to convert a CohortItem's billingPeriod in to the user friendly representation in letters
     // and emails.
-    "Month" -> "Monthly",
-    "Quarter" -> "Quarterly",
-    "Quarterly" -> "Quarterly",
-    "Semi_Annual" -> "Semiannually",
-    "Annual" -> "Annually"
+    "Month" -> "monthly",
+    "Quarter" -> "quarterly",
+    "Quarterly" -> "quarterly",
+    "Semi_Annual" -> "semiannually",
+    "Annual" -> "annually"
```

Comment:
what's the reason for changing the format here?

Reply:
@tomrf1 
Request from marketing. The change will affect all the migrations, but it was always wrong to start with. It means that all emails and letter will have a better capitalization from now on.